src/_P043_ClkOutput.ino

```diff
@@ -53,25 +55,20 @@ boolean Plugin_043(byte function, struct EventStruct *event, String& string)
 
     case PLUGIN_WEBFORM_LOAD:
       {
-        String options[3];
-        options[0] = "";
-        options[1] = F("Off");
-        options[2] = F("On");
-
+ 
         for (byte x = 0; x < PLUGIN_043_MAX_SETTINGS; x++)
         {
         	addFormTextBox(String(F("Day,Time ")) + (x + 1), String(F("p043_clock")) + (x), timeLong2String(ExtraTaskSettings.TaskDevicePluginConfigLong[x]), 32);
-//          addHtml(F("<TR><TD>Day,Time "));
-//          addHtml(x+1);
-//          addHtml(F(":<TD><input type='text' name='plugin_043_clock"));
-//          addHtml(x);
-//          addHtml(F("' value='"));
-//          addHtml(timeLong2String(ExtraTaskSettings.TaskDevicePluginConfigLong[x]));
-//          addHtml("'>");
-
-          addHtml(" ");
-          byte choice = ExtraTaskSettings.TaskDevicePluginConfig[x];
-          addSelector(String(F("p043_state")) + (x), 3, options, NULL, NULL, choice);
+          if (CONFIG_PIN1 >= 0) {
+            String options[3];
+            options[0] = "";
+            options[1] = F("Off");
+            options[2] = F("On");
```

Comment:
I do see 3 options here.
On/off I can understand but what does the "" mean?

Reply:
This is the code from the original version. You have 8 lines to define a schedule. So if you only need 2 times they will be defined on and off for example. The remaining lines will have "" and will not be executed. I volunteer to write the documentation of this plugin as well ;-)